pkg/arvo/sys/hoon.hoon

```diff
@@ -13412,6 +13417,9 @@
     ++  hunk  (most mush loan)                          ::  gapped specs
     ++  jump  ;~(pose leap:docs gap)                    ::  gap before docs
     ++  loaf  ?:(tol tall wide)                         ::  hoon
+    ++  loll  ?:  tol                                   ::  hoon without docs
+                tall:(vang bug wer |)
+              wide:(vang bug wer |)
```

Comment:
No need to call `+vang` again here, you can just use `tall(doc |)` or `wide(doc |)`.